.azure/gpu-tests.yml

```diff
@@ -75,7 +75,6 @@ jobs:
         CUDA_VERSION_MM=$(python -c "import torch ; print(''.join(map(str, torch.version.cuda.split('.')[:2])))")
         pip install "bagua-cuda$CUDA_VERSION_MM>=0.9.0"
         pip install -e .[strategies]
-        pip install deepspeed>0.6.4  # TODO: remove when docker images are upgraded
```

Comment:
If we don't have this, it will not install the latest deepspeed. Spongebob logic ...